src/posix/platform/radio.hpp

```diff
@@ -70,7 +83,30 @@ class Radio
     void ProcessRadioUrl(const RadioUrl &aRadioUrl);
     void ProcessMaxPowerTable(const RadioUrl &aRadioUrl);
 
-    RadioUrl mRadioUrl;
+    Spinel::SpinelInterface *CreateSpinelInterface(const char *aInterfaceName);
+
+    enum
```

Comment:
should we use `constexpr size_t` instead?

Reply:
Updated